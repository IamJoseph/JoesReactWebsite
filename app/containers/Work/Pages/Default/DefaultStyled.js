import styled from 'styled-components';

export default styled.img`
  color: #6d8da7;
  maxWidth: 300px;
  width: 80%;
  @media(min-width: 1000px){
    height: 65px;
  }
`;
