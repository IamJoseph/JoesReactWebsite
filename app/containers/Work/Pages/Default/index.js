import React from 'react';
import bluetoadLogo from '../../../../assets/bluetoad_logo.png';
import IMG from './DefaultStyled';
import { Container, UL, LI, H2 } from '../PagesStyled';

export default function Default() {
  return (
    <Container>
      <UL>
        <IMG src={bluetoadLogo} alt="bluetoad logo" />
        <div>July 2015 - Dec 2016</div>
        <div>
          <LI>
            Worked under pureMVC framework
          </LI>
          <LI>
            Dealt largely with cross domain communication
          </LI>
          <LI>
            Built unified metrics / analytics / GTM / thermostats API
          </LI>
          <LI>
            Created dynamic implementation of DFP
          </LI>
          <LI>
            Several UI overhauls
          </LI>
          <LI>
            Events refactor
          </LI>
          <LI>
            Colorbox
          </LI>
          <LI>
            SEO
          </LI>
          <LI>
            508 compliance
          </LI>
          <LI>
            AGILE w/ Jira
          </LI>
        </div>
      </UL>
      <UL>
        <H2 style={{ marginBottom: '.2em' }}>Web Development</H2>
        <div>Contract</div>
        <div>2014 - Present</div>
      </UL>
    </Container>
  );
}
