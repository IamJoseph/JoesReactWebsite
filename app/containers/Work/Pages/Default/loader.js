/**
 * Asynchronously loads the components for WorkPage
 */

import { errorLoading } from 'utils/asyncInjectors';

export default () => (cb) => {
  import('containers/Work/Pages/Default')
    .then(cb)
    .catch(errorLoading);
};
