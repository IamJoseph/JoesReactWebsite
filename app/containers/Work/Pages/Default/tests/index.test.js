import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Default from '../index';

describe('<Default />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(
      <Default />
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
