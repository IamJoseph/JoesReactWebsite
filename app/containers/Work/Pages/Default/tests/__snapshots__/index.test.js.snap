// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Default /> should render correctly 1`] = `
<styled.section>
  <styled.ul>
    <styled.img
      alt="bluetoad logo"
      src="IMAGE_MOCK"
    />
    <div>
      July 2015 - Dec 2016
    </div>
    <div>
      <styled.li>
        Worked under pureMVC framework
      </styled.li>
      <styled.li>
        Dealt largely with cross domain communication
      </styled.li>
      <styled.li>
        Built unified metrics / analytics / GTM / thermostats API
      </styled.li>
      <styled.li>
        Created dynamic implementation of DFP
      </styled.li>
      <styled.li>
        Several UI overhauls
      </styled.li>
      <styled.li>
        Events refactor
      </styled.li>
      <styled.li>
        Colorbox
      </styled.li>
      <styled.li>
        SEO
      </styled.li>
      <styled.li>
        508 compliance
      </styled.li>
      <styled.li>
        AGILE w/ Jira
      </styled.li>
    </div>
  </styled.ul>
  <styled.ul>
    <styled.h2
      style={
        Object {
          "marginBottom": ".2em",
        }
      }
    >
      Web Development
    </styled.h2>
    <div>
      Contract
    </div>
    <div>
      2014 - Present
    </div>
  </styled.ul>
</styled.section>
`;
